packages/cloud-cognitive/src/components/AddSelect/AddSelectSidebar.js

```diff
@@ -27,23 +40,32 @@ export let AddSelectSidebar = ({
           {multiSelection.length}
         </Tag>
       </div>
-      <div className={`${blockClass}-body`}>
-        {multiSelection.length > 0 ? (
-          multiSelection.map((item) => <p key={item}>{item}</p>)
-        ) : (
+      {multiSelection.length > 0 ? (
+        <Accordion align="start">
+          {sidebarItems.map((item) => (
+            <AccordionItem title={item.value} key={item.id}>
+              {Object.keys(item).map((key) => (
+                <p key={key}>{`${key} - ${item[key]}`}</p>
+              ))}
+            </AccordionItem>
+          ))}
+        </Accordion>
+      ) : (
+        <div className={`${blockClass}-body`}>
           <NoDataEmptyState
             subtitle={noSelectionDescription}
             title={noSelectionTitle}
             size="sm"
           />
-        )}
-      </div>
+        </div>
+      )}
     </div>
   );
 };
 
 AddSelectSidebar.propTypes = {
   influencerTitle: PropTypes.string,
+  items: PropTypes.array,
```

Comment:
It also might be worth extending the prop types for `items` a bit now that it looks like an `id` property is required

ie:
```jsx
items: PropTypes.arrayOf(
  PropTypes.shape({
    value: PropTypes.string,
    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
  })
)
```

I don't know if those are the correct prop types but giving a bit more depth as to the structure of `items` might be good.

Reply:
that's fair 👍